### DISCUSSION TITLE
Dịch symbolic tensor

### DISCUSSION THREAD

**Discussion by quangvnai** [mlbvn/handson-ml2-vn#134] [Terminology] [answered]
Từ này nên dịch là: `tensor tượng trưng` hay `tensor biểu tượng` nhỉ?

Context như sau:
```
symbolic tensor—a tensor without any actual value, only a name, a data type, and a shape.
```

**Comment by rootonchair** [comment, marked as answer]
Em nghiêng theo hướng `biểu tượng` do nó là danh từ, ko sợ xài nhầm

**Reply by quangvnai** [reply]
anh đồng ý, vậy chốt cách này nhé.
